cmd/broker/main.go

```diff
@@ -44,7 +45,9 @@ func main() {
 	fatalOnError(err)
 
 	srv := broker.New(sFact.Addon(), sFact.Chart(), sFact.InstanceOperation(), sFact.Instance(), sFact.InstanceBindData(),
-		bind.NewRenderer(), bind.NewResolver(clientset.CoreV1()), helmClient, log)
+		bind.NewRenderer(), bind.NewResolver(clientset.CoreV1()), helmClient, storageConfig.ExtractEtcdURL(), log)
```

Comment:
etcd parameter is not used anymore, remove from `internal/broker/server.go` and `internal/broker/broker.go`